source/monitoring/cluster-monitoring.txt

```diff
@@ -0,0 +1,136 @@
+implements the ``MongoDB\Driver\Monitoring\SDAMSubscriber`` interface,
+then use the ``MongoDB\Client::addSubscriber()`` method to register the
+event subscriber with your ``MongoDB\Client`` instance.
+
+The following code creates the ``MySubscriber`` class, which implements
+the ``SDAMSubscriber`` interface. The class is defined with a method to
+output a message when a ``ServerOpeningEvent`` is generated by the
+server:
+
+.. literalinclude:: /includes/monitoring/sdam.php
+   :start-after: start-mysubscriber
+   :end-before: end-mysubscriber
+   :language: php
+   :copyable:
+   :dedent:
+
+.. note::
+
+   As shown in the preceding code, you must implement all of the methods
+   of the ``SDAMSubscriber`` interface, even for events you are not subscribing to.
+   You can implement these methods with empty bodies so that the library
+   does not generate any messages for these events.
+
+Then, use the ``addSubscriber()`` method to register ``MySubscriber``
+with the client, as shown in the following code:
+
+.. literalinclude:: /includes/monitoring/sdam.php
+   :start-after: start-add-sub
+   :end-before: end-add-sub
+   :language: php
+   :copyable:
+   :dedent:
+
+When you run the application, your subscriber records the SDAM event and
+outputs messages such as the following:
+
+.. code-block:: none
+   :copyable: false
+
+   Server opening on ac-rmuag0v-shard-00-00.gh0qg50.mongodb.net:27017
+   Server opening on ac-rmuag0v-shard-00-01.gh0qg50.mongodb.net:27017
+   Server opening on ac-rmuag0v-shard-00-02.gh0qg50.mongodb.net:27017
+
+Event Descriptions
+------------------
+
+You can subscribe to the following SDAM events by implementing the
+corresponding method from the ``SDAMSubscriber`` interface:
+
+.. list-table::
+   :widths: 35 65 
+   :header-rows: 1
+
+   * - Event Name
+     - Description
+
+   * - ``ServerChangedEvent``
+     - Created when an instance state changes, such as from secondary to
+       primary.
+
+   * - ``ServerOpeningEvent``
+     - Created when the server is initialized.
+
+   * - ``ServerClosedEvent``
+     - Created when the server is closed.
+
+   * - ``TopologyChangedEvent``
+     - Created when the topology changes, such as an election of a new
+       primary or disconnection of a ``mongos`` proxy.
+
+   * - ``TopologyOpeningEvent``
+     - Created when the topology is initialized.
+
+   * - ``TopologyClosedEvent``
+     - Created when the topology is closed.
+
+   * - ``ServerHeartbeatStartedEvent``
+     - Created when the heartbeat is started.
+
+   * - ``ServerHeartbeatSucceededEvent``
+     - Created when the heartbeat succeeds.
+
+   * - ``ServerHeartbeatFailedEvent``
+     - Created when the heartbeat fails.
+
+API Documentation
+-----------------
+
+To learn more about any of the classes or methods discussed in this guide, see the
+following API documentation:
+
+- :phpmethod:`MongoDB\Client::addSubscriber()`
+- :phpclass:`MongoDB\Client`
```

Comment:
The `removeSubscriber()` method would be a more helpful reference. I don't see any reason to link to the Client class page directly.

On that note, perhaps you should include a brief example of unregistering a subscriber.